### DISCUSSION TITLE
Assets upload to S3 Digital Ocean return 400 Error

### DISCUSSION THREAD

**Discussion by zimiovid** [outline/outline#6619] [Self Hosting]
Good day! When uploading attachments through document editing, we receive a 400 error. The attachments are uploaded to our s3 based on Digital Ocean.

**To Reproduce**
1. Go to  any document
2. Click on Edit
3. Try insert image and await upload to s3
4. See error in console

**Expected behavior**
The attachments should upload to s3 without problems.

**Screenshots**
Error in Browser console:
![image](https://github.com/outline/outline/assets/61754500/3b16aaca-8977-48c7-9e9f-c20986f8844f)
![image](https://github.com/outline/outline/assets/61754500/5946cfed-de06-4dc9-886b-6f7aea3543a8)

Recorded JAM https://jam.dev/c/9cdb19e0-69ff-4a52-886f-8b4539c520b3

**Outline (please complete the following information):**
- Install: self hosted
- Version: 0.75.2

**Desktop (please complete the following information):**
 - OS: Linux, Windows, MacOS
 - Browser: Chrome
 - Version: Last versions

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-290/assets-upload-to-s3-digital-ocean-return-400-error">OLN-290 Assets upload to S3 Digital Ocean return 400 Error</a></p>
